.github/workflows/psammead-publish.yml

```diff
@@ -22,10 +22,10 @@ jobs:
           scope: '@bbc'
 
       - name: Install Node Modules
-        run: npm run ci:packages
+        run: yarn run ci:packages
```

Comment:
It's small and doesn't make much different but I think with yarn you can also remove the `run` command so just `yarn ci:packages` should work.

Not crucial to this work though.

Reply:
Removed `run` from all yarn commands in [this commit](https://github.com/bbc/psammead/pull/4335/commits/a335bd91d50a5808b9a685b3080385b4b6f5ffb1)